tracing/tracing.go

```diff
@@ -23,7 +23,7 @@ const (
 )
 
 var (
-	Tracer = trace.NewNoopTracerProvider().Tracer("noop")
+	Tracer = noop.NewTracerProvider().Tracer("noop")
```

Comment:
I ran `staticcheck ./...` to check unused fields that might've been left from the refactor. This was the only complaint, since it's one line I figured I'd add it.